packages/design-system/src/components/Dialog/Dialog.tsx

```diff
@@ -1,194 +1,19 @@
 import { EVENT_CATEGORY, MAX_LENGTH, sendLinkEvent } from '../analytics/SendAnalytics';
-import AriaModal from 'react-aria-modal';
 import Button, { ButtonVariation } from '../Button/Button';
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useLayoutEffect, useRef } from 'react';
 import classNames from 'classnames';
 import { dialogSendsAnalytics } from '../flags';
 import { CloseIcon } from '../Icons';
+import dialogPolyfill from 'dialog-polyfill';
+import 'dialog-polyfill/dist/dialog-polyfill.css';
```

Comment:
Ideally, I'd like to conditionally import this CSS if `window.HTMLDialogElement` doesn't exist, but ran into issues getting that to work. Importing it here just to demo what an IE solution would look like.

Reply:
Also pointing out that adding the polyfill CSS at the top doesn't _hurt_ our dialog. So maybe this is fine